ambry-api/src/main/java/com/github/ambry/config/RouterConfig.java

```diff
@@ -234,6 +236,20 @@ public class RouterConfig {
   @Default("2")
   public final int routerPutSuccessTarget;
 
+  /**
+   * The maximum number of parallel requests issued at a time by the ReplicateBlob manager.
+   */
+  @Config(ROUTER_REPLICATE_BLOB_REQUEST_PARALLELISM)
+  @Default("3")
```

Comment:
confused. Why are we issuing 3 replicateBlob requests ? 
If a ttl-update fails due to 1 missing replica , then shdn't we just issue that. Basically, we shd issue the minimum number of replicateBlob requests just enough for the ttl-update or whatever other mutation to succeed.